gwpy/table/gravityspy.py

```diff
@@ -19,6 +19,10 @@
 """Extend :mod:`astropy.table` with the `GravitySpyTable`
 """
 
+import os.path
+
+from six.moves.urllib.request import urlopen
```

Comment:
@duncanmmacleod, will these changes be part of a future (python2.7-compliant) 1.x-series release?

Reply:
Yes, these are marked for the 1.0.1 milestone.